File: Voronezh/Bars.Gkh.Regions.Voronezh/libs/B4/model/RosRegExtractGov.js
Ext.define('B4.model.RosRegExtractGov', {
    extend: 'B4.base.Model',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'RosRegExtractGov'
    },
    fields: [
        { name: 'Id' },
        { name: 'Gov_Code_SP' },
        { name: 'Gov_Content' },
        { name: 'Gov_Name' },
        { name: 'Gov_OKATO_Code' },
        { name: 'Gov_Country' },
        { name: 'Gov_Address' }
    ]
});

File: Voronezh/Bars.Gkh.Regions.Voronezh/Entities/RosRegExtract/RosRegExtractGov.cs
using Bars.B4.DataAccess;
using Bars.Gkh.Entities;

namespace Bars.Gkh.Regions.Voronezh.Entities
{
    public class RosRegExtractGov : PersistentObject
    {
        //Governance
        public virtual string Gov_Code_SP { get; set; }
        public virtual string Gov_Content { get; set; }
        public virtual string Gov_Name { get; set; }
        public virtual string Gov_OKATO_Code { get; set; }
        public virtual string Gov_Country { get; set; }
        public virtual string Gov_Address { get; set; }
    }
}
